packages/trpc/server/routers/viewer/workflows.tsx

```diff
@@ -413,6 +416,19 @@ export const workflowsRouter = router({
         }
       });
 
+      for (const activeRef of workflowToDelete.activeOn) {
```

Comment:
Oh, I missed this. I thought I had handled deletion.
But we should use `removeSmsReminderFieldForBooking` as that handles these hardcoded values.

Reply:
Thank you, fixed 👍